### DISCUSSION TITLE
Is "new DateTime(0, 0, 0);" intentional?

### DISCUSSION THREAD

**Discussion by sygmond** [fullstackhero/blazor-starter-kit#366] [Q&A]
https://github.com/blazorhero/CleanArchitecture/blob/45d6e675750a5138b40aa3a3c42bc89c431442a6/src/Client/Shared/Components/AddEditExtendedAttributeModal.razor.cs#L59

Hi. First of all, great work! Amazing work! Thank you for your hard work into writing this beautiful template!

Is DateTime supposed to have 0 as values in here? As far as I know, this will throw an exception. Is this intentional? Thank you!

**Comment by ajakblackgoat** [comment]
I dont think so. Tested with ```DateTime dt = new DateTime(0, 0, 0);``` results in runtime _ArgumentOutOfRangeException : Year, Month, and Day parameters describe an un-representable DateTime._
